docs/pages/docs/apis/access-control.mdx

```diff
@@ -20,81 +20,74 @@ export default config({
 
-The examples below will all use the concise syntax, however the various access control rules can all be applied using verbose syntax.
+If access is **denied** due to any of the access control methods, the following responses will be returned from the GraphQL API:
 
-There are three different ways you can specify access-control rules: **static**, **declarative**, and **imperative**.
+- **Mutations**
+  - **Single** operations return `null` and an access denied error.
+  - **Multi** operations return a data array with `null` values for the items which have access denied.
+    Error responses are returned for each `null` item.
+- **Queries**
+  - **Single** item queries return `null` with no errors.
+  - **Many** item queries filter out items which have access denied. No error responses are returned.
+  - **Count** queries will only count those items for which access is **not** denied. No error responses are returned.
 
-### Static (list)
+### Operation
 
-Static access control rules are simple boolean values.
-A value of `true` indicates that all users can perform the operation.
-A value of `false` indicates that no users can perform the operation.
+Operation-level access control lets you control whether which operations can be accessed by a user based on the `session` and `context`.
+Individual functions can be provided for each of the operations.
 
-A static value of `false` implies that the operation can never be executed.
-As such, Keystone will exclude the related operations and types from the GraphQL API.
-For example, if you set `{ create: false }` then the mutations `createItem` and `createItems` will be removed from the GraphQL API.
-If you want to keep the operations in the GraphQL API while preventing all access, you can use the [imperative](#imperative-list) access control rule `() => false`.
-The excluded operations can still be access by using [`context.sudo()`](./context#new-context-creators).
+!> These functions must return `true` if the operation is allowed, or `false` if it is not allowed.
 
 ```typescript
 import { config, createSchema, list } from '@keystone-next/keystone';
 
 export default config({
   lists: createSchema({
     ListKey: list({
-      // Static access control definition
-      access: true,
+      access: {
+        operation: {
+          query: ({ session, context, listKey, operation }) => true,
+          create: ({ session, context, listKey, operation }) => true,
+          update: ({ session, context, listKey, operation }) => true,
+          delete: ({ session, context, listKey, operation }) => true,
+        }
+      },
     }),
   }),
 });
 ```
 
-### Declarative (list)
+?> The `query` access control rule is applied when running GraphQL query operations.
+It does not prevent a user **reading** the data returned by the mutation operations.
```

Comment:
might be good to explicitly say how someone _can_ prevent a user reading data: to prevent a user from reading an item, you need to have access control rules for create, update and delete as well as query